Directory.Build.props

```diff
@@ -1,5 +1,8 @@
 <Project>
   <PropertyGroup>
+    <!-- When this changes, the .NET runtime installation
+         at https://github.com/Azure/azure-mcp/blob/e98c1919a25cc1e4e9325446d0ff7aec098fca67/eng/pipelines/templates/jobs/build.yml#L39
+         may need to be updated. -->
```

Comment:
I read the referenced build.yml file but was unable to understand the intent of this code comment. Could we please clarify its meaning?

Reply:
When you install an SDK, that SDK only comes with the .NET runtime matching it. So when the global.json moved to 10p6 .NET SDK, it came with the .NET 10 runtime. Since your MCP server and tests _target_ .NET 9, we need to ensure that a .NET 9 runtime is installed on the build agents. This is a fairly common pattern across .NET repositories - the SDK tooling doesn't force you to use a particular .NET Runtime, but you do need to ensure it's available.